docs/platform/longview/what-is-longview/index.md

```diff
@@ -5,15 +5,14 @@ author:
 description: 'This guide shows how to install and use Linode Longview. Longview is Linode’s system data graphing service. It tracks metrics for CPU, memory, and network bandwidth, both aggregate and per-process, and it provides real-time graphs that can help expose performance problems.'
 keywords: ["system monitoring", "longview", "metrics"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-aliases: ['platform/longview/longview/']
+aliases: ['platform/longview/longview/','longview/', 'uptime/longview/']
```

Comment:
Should this include the classic manager alias as well?

Reply:
No, because the `longview-classic-manager` never existed as a guide. Classic manager version before the cloud manager version was created was: `platform/longview/longview/`.